lib/force.go

```diff
@@ -1153,7 +1153,7 @@ func (f *Force) _coerceHttpError(res *http.Response, body []byte) error {
 	} else {
 		var errors ForceErrors
 		if err := internal.JsonUnmarshal(body, &errors); err != nil && !sessionExpired {
-			return err
+			return fmt.Errorf("unhandled error: %d %s", res.StatusCode, string(body))
```

Comment:
Hmmm should this be `return err` but we should have an `else` that does this?

Reply:
I'm not sure what you mean.  If we return `err` here, it's the error from failing to unmarshal the body.

Reply:
Oh, sorry, I didn't realize that we got rid of the "try to unmarshal into a map". This makes sense now!